memo/README.md

```diff
@@ -0,0 +1,4 @@
+# Memo Program
+
+A simple program that accepts a string of encoded characters and verifies that
+it parses. Currently handles UTF-8.
```

Comment:
Maybe a little more here that describes the instructions that the program supports?